lib/internal/modules/esm/public_loader_proxy.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
```

Comment:
This can be merged into `loader.js` too if you wrap the side-effect below into a function that produces a loader proxy and use a boolean to make sure it only gets created once. That reduces one more `require()` call. (In general, avoid doing side-effects at the top-level in internal modules, that makes the modules themselves unsnapshottable. Most things can be done in functions that get exported and called on-demand. It's an overkill to rely on the internal module loader to implement call-only-once, that can be easily achieved with a `if (called) { return; } else { called = true; }` pattern.